src/editor/EditorManager.js

```diff
@@ -91,7 +91,7 @@ define(function (require, exports, module) {
      * @private
      */
     var _inlineDocsProviders = [];
-
+    var _inlineEditProviderCheck = [];
```

Comment:
I'm confused about this.  Why do you have an array?  We're going to want a single function we can call that looks at the current editor + position and sees if any editor provider can give us info.